src/main/java/commands/WorkoutCommand.java

```diff
@@ -69,19 +83,26 @@ public void execute() {
         try {
             switch (getUserAction()) {
             case CREATE_ACTION_KEYWORD:
-                Workout newWorkout = workoutList.createAndAddWorkout(getUserArguments());
-                ui.printNewWorkoutCreatedMessage(newWorkout);
+                Workout newWorkout = getWorkoutList().createAndAddWorkout(getUserArguments());
+                getUI().printNewWorkoutCreatedMessage(newWorkout);
+                fileManager.writeNewWorkoutToFile(newWorkout);
                 break;
             case LIST_ACTION_KEYWORD:
-                workoutList.listWorkout();
+                getWorkoutList().listWorkout();
                 break;
             case DELETE_ACTION_KEYWORD:
                 Workout deletedWorkout = workoutList.deleteWorkout(getUserArguments());
                 ui.printDeleteWorkoutMessage(deletedWorkout);
```

Comment:
For standardization with the rest of the switch code:

Workout deleteWorkout = getWorkoutList().deleteWorkout(getUserArguments());
getUI().printDeleteWorkoutMessage(deletedWorkout);

Reply:
Thanks for spotting this. I was creating some accessor methods and missed out on some of them.

Changed in Commit ce06262